tests/units/iop/test_iop_trigger_files_processing.py

```diff
@@ -0,0 +1,53 @@
+from unittest.mock import patch
+
+import pytest
+from airflow.models import DagBag
+from airflow.models.dagrun import DagRun
+
+
+class S3BucketResultObj:
+    key: str
```

Comment:
why do you need this?

Reply:
just to have an object, which mimics the s3 object (which has a key). For example, we need it when we are searching for all extracted articles: https://github.com/cern-sis/workflows/blob/main/dags/iop/repository.py#L55

Reply:
hmm what do you mean by object? Do you need an attribute, a property, a method? 

```
class Hello:
     def __init__(self, key):
         self.key = key

hey = Hello('this is the key')
hey.key
> 'this is the key
```